src/main/java/com/amplitude/api/AmplitudeClient.java

```diff
@@ -1640,14 +1682,14 @@ public void identify(Identify identify) {
      * @param identify an {@link Identify} object
      * @param outOfSession whther to log the identify event out of session
      */
-    public void identify(Identify identify, boolean outOfSession) {
+    public void identify(Identify identify, boolean outOfSession, MiddlewareExtra extra) {
```

Comment:
Can you also please add extra to `setUserProperties()` we are using it in codegen since it's more concise than constructing an Identfy object from JSON.
https://github.com/amplitude/javascript/pull/13541/files#diff-fba79090504d6bb2a35fa69573380efd599f48f048d825c60c6ada66375c2305R246